src/multi-selector.js

```diff
@@ -79,32 +80,49 @@ const MultiSelector = React.createClass({
 			activeIndex: index,
 			mouseIndex: null,
 		}, () => {
-			this.searchItems[this.state.activeIndex].scrollIntoView();
-			if (this.state.mouseActive) {
-				this.setState({
-					mouseActive: false,
-					mouseFunc: () => {
-						this.setState({
-							mouseActive: true
-						})
-						document.removeEventListener("mousemove", this.state.mouseFunc);
-					}
-				}, () => {
-					document.addEventListener("mousemove", this.state.mouseFunc);
-				})
+			if (!isNull(this.state.activeIndex) && !isNull(this.searchItems[this.state.activeIndex])) {
+				this.searchItems[this.state.activeIndex].scrollIntoView();
+				if (this.state.mouseActive) {
+					this.setState({
+						mouseActive: false,
+						mouseFunc: () => {
+							this.setState({
+								mouseActive: true
+							})
+							document.removeEventListener("mousemove", this.state.mouseFunc);
+						}
+					}, () => {
+						document.addEventListener("mousemove", this.state.mouseFunc);
+					})
+				}
 			}
 		})
 	},
+	keyUp: function(e) {
+		const filterItems = this.getFilterItems(this.props.items);
+		const activeIndex = this.state.activeIndex;
+		this.props.onInputChange && this.props.onInputChange(e.currentTarget.value);
```

Comment:
I haven't seen any usability weirdness but yeah technically the input changes on `keyDown` so I think i'll move this back